test/e2e/multi-cluster-deployment/run-test.sh

```diff
@@ -18,25 +18,43 @@
 
 set -x # so users can see what is going on
 
+env="kind"
+
 if [ "$1" == "--released" ]; then
     setup_flags="$1"
     shift
 fi
 
-if [ "$#" != 0 ]; then
-    echo "Usage: $0 [--released]" >& 2
-    exit 1
+if [ "$1" == "--env" ]; then
+    env="$2"
+    shift
 fi
 
+# if [ "$#" != 0 ]; then
+#     echo "Usage: $0 [--released]" >& 2
+#     exit 1
+# fi
+
 set -e # exit on error
 
-SRC_DIR="$(cd $(dirname "${BASH_SOURCE[0]}") && pwd)"
-COMMON_SRCS="${SRC_DIR}/../common"
-HACK_DIR="${SRC_DIR}/../../../hack"
+if [ $env == "kind" ];then
+    SRC_DIR="$(cd $(dirname "${BASH_SOURCE[0]}") && pwd)"
+    COMMON_SRCS="${SRC_DIR}/../common"
+    HACK_DIR="${SRC_DIR}/../../../hack"
+
+    "${HACK_DIR}/check_pre_req.sh" --assert --verbose kubectl docker kind make go ko yq helm kflex ocm
 
-"${HACK_DIR}/check_pre_req.sh" --assert --verbose kubectl docker kind make go ko yq helm kflex ocm
+    "${COMMON_SRCS}/cleanup.sh"
+    source "${COMMON_SRCS}/setup-shell.sh"
+    "${COMMON_SRCS}/setup-kubestellar.sh" $setup_flags
+    "${SRC_DIR}/use-kubestellar.sh"
+fi
+
+
+if [ $env == "ocp" ];then
+   bash <(curl -s https://raw.githubusercontent.com/kubestellar/kubestellar/release-$KUBESTELLAR_VERSION/test/e2e/common/cleanup.sh) --env ocp
```

Comment:
Why the change to fetching the cleanup script from github? In the `kind` case, the local copy is used. These E2E tests are specifically designed to be able to test the local copy of the repo.

Reply:
This e2e test is focused to provide support to test KS releases on OCP - not a local copy.

Reply:
@dumb0002 - correct. The testing that supports the on going development (i.e., tests that run on latest main for PR validation) are planed to be running only through the automatic github actions. 
Once we have the OCP tests hooked into the automatic CICD and decide we want to have OCP validation also for the ongoing development we can improve the tests to supprt that as well.

Reply:
@dumb0002 : by "local copy" I mean the code copy, not where the containers run. The E2E tests have always been able to test a contributor's modified version of KubeStellar. This PR breaks that.